app/models/filter.rb

```diff
@@ -123,15 +127,15 @@ def granular?
     end
   end
 
-  def allows_taxonomies_filtering?
-    allows_organization_filtering? || allows_location_filtering?
+  def resource_taxable?
+    resource_taxable_by_organization? || resource_taxable_by_location?
   end
 
-  def allows_organization_filtering?
+  def resource_taxable_by_organization?
     granular? && resource_class.allows_organization_filtering?
   end
 
-  def allows_location_filtering?
+  def resource_taxable_by_location?
     granular? && resource_class.allows_location_filtering?
   end
```

Comment:
This is a public API. Is it used by any plugins?

Reply:
Public method of a class is IMHO not a public API.

This method is defined in Authorizable module and this specific class overrides it's behavior. The method seems to be used e.g. in `app/views/filters/_form.html.erb` to decide what to show in the form. This way, it would start showing the form part for everything, ignoring the resource of the filter. If we decide to refactor to avoid the meaning misunderstanding, we need to modify it there too. We also use these methods in audit extensions, where they IMHO need to remain as is. The idea is, the filter taxonomies are delegated to the resource class of the filter.

Reply:
Even if we don't consider it a public API, I am still worried about a plugin using it. A quick grep might be sufficient to check.

Reply:
It is not used in any plugin in theforeman org and it should not be, I'm keeping the method from Taxonomix used by Authorizer intact, these are just methods on `Filter`.